distribution/lib/Standard/Table/0.0.0-dev/src/Excel/Excel_Format.enso

```diff
@@ -0,0 +1,63 @@
+from Standard.Base import all
+
+import Standard.Table.Data.Table
+
+from Standard.Table.Excel.Section import Excel_Section, Worksheet, Sheet_Names, Range_Names
+from Standard.Table.Excel.Reader import read_excel
+from Standard.Table.Excel.Writer import write_excel
+
+## PRIVATE
+   Resolve the xls_format setting to a boolean.
+is_xls_format : (Boolean|Infer) -> File -> Boolean
```

Comment:
Nitpick but `is_xls_format True (File.new "my_TEXT_file.txt") == True` - looks a bit confusing - wait 'txt' is not XLS.
Maybe we can call it `should_treat_as_xls_format`? Then it will express the intent a bit clearer IMO.